doc/languages.rst

```diff
@@ -115,55 +179,99 @@ Programming languages
 Template languages
 ------------------
 
+* Angular2 templates
```

Comment:
Angular has no (meaningful) version number any more, should be `Angular templates`. Angular 2 was the big move from AngularJS to "Angular".

Reply:
ok. I don't know anything about Javascript stuff :)

Reply:
I unfortunately know more about this than I wanted to know :)